src/server/routes/summary/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+const { entitySearch } = require('./entity');
+
+router.get('/entity/search', function (req, res) {
+  let tokens = req.query.q.trim().split(' ');
+  entitySearch( tokens )
+    .then( r => res.json( r ) )
+    .catch( e => res.status( 500 ).send( { "error": "Server error" } ));  // eslint-disable-line no-unused-vars
```

Comment:
I don't think JSON makes sense in the case of an error.  It's going to be plain text.

Reply:
Returning text.